lib/DBICx/Sugar.pm

```diff
@@ -95,23 +96,21 @@ sub schema {
             $schema = $schema_class->connect( @conn_info );
         }
     } else {
-        my $dbic_loader = 'DBIx::Class::Schema::Loader';
-        eval { load $dbic_loader };
-        die "You must provide a schema_class option or install $dbic_loader."
-            if $@;
-        $dbic_loader->naming( $options->{schema_loader_naming} || 'v7' );
+        eval { require DBIx::Class::Schema::Loader; 1; }
```

Comment:
The `eval` is a bit clearer and I always prefer using barewords for `require` where possible.

I changed it because I hoped it will be clearer for others to read. This was missed in a comment on Dancer2-Plugin-DBIC in which recognizing this was used was missed.

Feel free to skip this if you don't like it.